terraform/dev/main.tf

```diff
@@ -33,6 +33,11 @@ data "sops_file" "secrets" {
 locals {
   env     = "dev"
   secrets = jsondecode(data.sops_file.secrets.raw)
+  data_producer_allowed_ips = [
```

Comment:
Can we put these IPs in the sops secret files? Also these IPs appear to be the public IPs for the BODS frontend (load balancer), we'll need to whitelist the elastic IPs mapped to the NAT gateways since they'll be the IPs associated with any outbound traffic from the BODS backend (I'm just assuming this since I don't have access to the BODS AWS account to check)